File: Unit4A.ts
/**
 * @author William Provost
 * @version 1.0.0
 * @date 2025-11-24
 * @fileoverview This program counts backward by 5's from 100 down to 0.
 */

// variable
let currentValue: number = 100;

// output header
console.log("Backward count by 5's from 100 to 0:");

// loop to count backward by 5's
while (currentValue >= 0) {
  console.log(currentValue);
  currentValue -= 5;
}

console.log("\nDone.");

File: Unit4A.go
// Author: William Provost
// Version: 1.0.0
// Date: 2025-11-25
// Fileoverview: This program counts backward by 5's from 100 down to 0.

package main

import "fmt"

func main() {
	currentValue := 100

	fmt.Println("Backward count by 5's from 100 to 0:")

	for currentValue >= 0 {
		fmt.Println(currentValue)
		currentValue -= 5
	}

	fmt.Println("\nDone.")
}
